Gemfile

```diff
@@ -60,6 +60,7 @@ gem 'shell-spinner', '~> 1.0', '>= 1.0.4'
 gem 'ruby-progressbar'
 gem 'geckoboard-ruby'
 gem 'posix-spawn', '~> 0.3.13'
+gem 'rack-protection',         '~> 1.5.4'
```

Comment:
**[tiny nit pick]** ✂️  indentation (at least doesn't seem to match the one used just above it 😰